haas/api.py

```diff
@@ -822,6 +813,9 @@ def show_node(nodename):
         'free': node.project_id is None,
         'nics': [{'label': n.label,
                   'macaddr': n.mac_addr,
+                  'networks': dict([(attachment.channel,
```

Comment:
I understand it's not required to. As a human interactive user, it's just easier for me to work with a list that is sorted; especially in operations where I think something is connected a certain way and I just want to verify that.